.github/workflows/build.yml

```diff
@@ -44,6 +42,11 @@ jobs:
           key: ${{ runner.os }}-gradle-${{ hashFiles('*.gradle.kts') }}
           restore-keys: |
             ${{ runner.os }}-gradle-
+      # Github runner uses inconsistent naming for the watch simulators, create a simulator that matches what the tests expect
+      - name: Create watch simulator
+        if: matrix.os == 'macOS-latest'
+        shell: bash
+        run: xcrun simctl create "Apple Watch Series 5 - 44mm" "Apple Watch Series 5 (44mm)"
```

Comment:
I've seen other folks solve this by forcing Xcode 13 instead. In Kotlin 1.8 it should be fixed by reading the actual device list instead of hardcoding it https://youtrack.jetbrains.com/issue/KT-54090/Take-an-Apple-test-device-from-the-device-list

Reply:
Is there a preference? This doesn't seem to cost much and lets us keep testing on the default xcode

Reply:
no preference. I have no idea if one is faster or anything, though I suspect it's minor either way. But that comment is slightly inaccurate because the issue is a change from xcode, not github